sc-kpm/sc-python/services/http_api/auth/database.py

```diff
@@ -0,0 +1,84 @@
+from sqlalchemy import Column, Integer, String
```

Comment:
Please remove users from DB except single admin user. Maybe last user also should be renamed to admin